### PR TITLE
Added basic signup/login handling with views.

### PR DESCRIPTION
**Pull request by smith1302**
Added basic signup/login handling with views. Also added some structure to our project, namely a constants file so we can add things in there as we go.

### CODE REVIEW

File: ParseStarterProject/AppDelegate.swift [hunk 1]
```diff
@@ -45,7 +45,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // PFFacebookUtils.initializeFacebook()
         // ****************************************************************************
 
-        PFUser.enableAutomaticUser()
+        // PFUser.enableAutomaticUser()
```
Comment (smith1302) [inline ParseStarterProject/AppDelegate.swift:48]: Needed to get rid of this line. It allowed for anonymous login (so the user is always logged in)

File: ParseStarterProject/Constants.swift [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import UIKit
+
+class Constants {
+    static let appName = "Selfiesteem"
+    static let storyboard = UIStoryboard(name: "Main", bundle: nil) //  Get storyboars singleton instance
+}
```
Comment (smith1302) [inline ParseStarterProject/Constants.swift:6]: Whenever we need to reusable global constants lets throw them in here.

File: ParseStarterProject/IntroViewController.swift [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    }
+    
+    /*
+    * User is signed up or logged in. Lets take them to the main app
+    */
+    func continueToMainApp() {
+        //let storyboard = Constants.storyboard
+        //let mainAppVC = storyboard.instantiateViewControllerWithIdentifier("nameOfVCGoesHere")
+        //self.presentViewController(mainAppVC, animated: true, completion: nil)
+    }
```
Comment (smith1302) [inline ParseStarterProject/IntroViewController.swift:112]: Not good practice to push code that is commented out, but I didn't have the next view controller set up yet and wanted to include this logic in this commit.

File: ParseUI.framework/Headers/PFCollectionViewCell.h [hunk 4]
```diff
@@ -0,0 +1,61 @@
+/*
```
Comment (smith1302) [inline ParseUI.framework/Headers/PFCollectionViewCell.h:1]: Anything from this file and below is auto generated code added when I included the parseUI framework. Ignore this

File: ParseStarterProject/ViewController.swift [hunk 5]
```diff
@@ -1,24 +0,0 @@
...
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
-    }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
-    }
-}
```
Comment (smith1302) [inline ParseStarterProject/ViewController.swift:24]: This was in the boiler plate. We don't need this

File: ParseStarterProject/IntroViewController.swift [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    
+    // Instantiate our login and sign up views
+    var loginViewController: PFLogInViewController = PFLogInViewController()
+    var signUpViewController: PFSignUpViewController = PFSignUpViewController()
+    
+    /*
+    * When the view appears, we need to check if the user is already logged in. If they are let's skip this screen
+    * and go straight to the main app.
+    */
+    override func viewDidAppear(animated: Bool) {
```
Comment (dyeung93) [inline ParseStarterProject/IntroViewController.swift:24]: Making an assumption that this function below is for returning users

Comment (smith1302) [inline ParseStarterProject/IntroViewController.swift:24]: The PFUser.currentUser() line? Yeah it's parse's way of returning the user from the current session if there is one

**Pull request merged**